update.sh

```diff
@@ -15,6 +15,10 @@ git pull origin main
 git checkout main
 cd ..
 
+echo "update commit-msg"
+cp .github/hooks/commit-msg .git/hooks/commit-msg
+chmod +x .git/hooks/commit-msg
```

Comment:
This should be changed to follow above paths.

Reply:
this is fixed on the last commit